packages/flutter/test/material/scaffold_test.dart

```diff
@@ -574,6 +574,62 @@ void main() {
       expect(tester.element(find.byKey(testKey)).size, const Size(88.0, 48.0));
       expect(tester.renderObject<RenderBox>(find.byKey(testKey)).localToGlobal(Offset.zero), const Offset(0.0, 0.0));
     });
+
+    WidgetTesterCallback getBodySizeTesterCallback({bool expanded = false, Size size}) => (WidgetTester tester) async {
+      final Key bodyKey = UniqueKey();
+
+      Widget buildFrame(bool extendedBody) {
+        return MaterialApp(
+          home: Scaffold(
+            extendedBody: extendedBody,
+            body: Container(
+              key: bodyKey,),
+            bottomNavigationBar: BottomAppBar(
+              child: Container(height: 48.0,),
+            ),
+          ),
+        );
+      }
+
+      await tester.pumpWidget(buildFrame(expanded));
```

Comment:
expanded => extendedBody ... but see earlier comment about restructuring this test and the following one.